project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -119,6 +144,21 @@ function c4m_features_og_group_dashboard_action_promote($node) {
 }
 
 /**
+ * Callback to show the recomendation link.
+ *
+ * @param object $node
+ *    The node object.
+ *
+ * @return string
+ *    Rendered output of the promote link.
+ */
+function c4m_features_og_group_dashboard_action_recommend($node) {
+  $flag_link = flag_create_link('c4m_og_content_recommend', $node->nid);
+  $output = '<div class="group-actions-recommend">' . $flag_link . '</div>';
```

Comment:
Could you use [theme_html_tag](https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme_html_tag/7.x) here please.